src/styles/blocks/product-card.scss

```
@@ -0,0 +1,96 @@
+.card {
+  justify-content: space-between;
+  box-sizing: border-box;
+  max-width: 200px;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-size: 10px;
+  font-weight: 400;
+  font-style: normal;
+
+  &__pic {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__name,
+  &__id,
+  &__ranking,
+  &__price-block,
+  &__buy-button {
+    margin: 0 16px 16px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 0;
+  }
+
+  &__id {
+    margin-top: 4px;
+    line-height: 14px;
+    color: $sec-color-grey;
+    font-weight: 400;
+    font-size: 10px;
+  }
+
+  &__ranking {
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    line-height: 14px;
+  }
+
+  &__price-block {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-name {
+    font-weight: 400;
+    line-height: 18px;
+    font-size: 12px;
+    color: $sec-color-grey;
+  }
+
+  &__price-num {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__buy-button {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    border: 1px transparent $highlight-color;
+    background: $highlight-color;
+    border-radius: 5px;
+    transition-duration: 0.6s;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: #fff;
+
+    &:hover {
+      border: 1px solid $highlight-color;
```

Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover
```suggestion
      border-color: $highlight-color;
```